test/workers/callbacks/http_callback_test.rb

```diff
@@ -23,7 +23,7 @@ class HttpCallbackTest < ActiveSupport::TestCase
   it 'can call an http callback' do
     stub_request(:post, 'https://thisisatest.com/callback').
       with(body: '{"result":"success"}',
-           headers: { 'Content-Type' => 'application/json; charset=utf-8', 'Host' => 'thisisatest.com:443' } ).
+           headers: { 'Content-Type' => 'application/json; charset=utf-8', 'Host' => 'thisisatest.com' } ).
```

Comment:
Line is too long. [107/80]<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.